packages/language-server/src/completion/completions.json

```diff
@@ -65,6 +65,10 @@
     {
       "label": "binaryTargets",
       "documentation": "Specifies the OS on which the Prisma Client will run to ensure binary compatibility of the query engine."
+    },
+    {
+      "label": "previewFeatures",
+      "documentation": "Enables feature flags. This enables us to hide functionality from the *default* API that *all* users get when installing Prisma and Prisma Client. Instead, users can *explicitly* opt-in to certain features by enabling them in this field."
```

Comment:
I think this should be written more from the user perspective, what it _gives_ them if they enable a Preview Feature Flag.

Reply:
How about `Enables preview features fresh from our developers for you to try out.`